Add-ons/Starter-Kit-TEMP/Widgets/index.md

```diff
@@ -260,3 +260,27 @@ Part two is for adding logo and settings for these. For each logo you want to ad
 
 Below you can see an example that uses the back and white feature, and has two logos.
 ![movePage.jpg](images/Logos.png)
+
+
+### The Global Content Widget
+![movePage.jpg](images/The-Global-Content-Widget.png)
+
+What The Global Content Widget does for you, is that it allows you to pick those templates you created, and use them on your pages.
```

Comment:
If you read about the Global Content Widget before you know what Global Content is, I think we need to rephrase this first sentence a bit:

```suggestion
What this widget does for you, is that it allows you to pick pre-made widgets, also called Global Content.

To learn more about how to create Global Content, you can read the [Global Content](../Content-Types/Global-Content/index.md) article.
```